libs/auth-api-lib/seeders/data/client-scope-user-profile-notifications.ts

```diff
@@ -0,0 +1,17 @@
+import { compose, createClient, createScope } from './helpers'
+
+export const up = compose(
+  createClient({
+    clientId: '@island.is/system/notifications',
+    clientType: 'machine',
+    displayName: 'Notifications machine client',
+    description: '',
+    allowedScopes: ['@island.is/user-profile:admin'],
```

Comment:
This will break because the scope doesn't exist yet. addToClient below is enough.

Reply:
so empty array at that point => "allowedScopes: []," ?

Reply:
Yeah. I guess you can just the field.